services/console-proxy/server/src/main/java/com/cloud/consoleproxy/ConsoleProxy.java

```diff
@@ -76,6 +76,7 @@ public class ConsoleProxy {
     static int httpCmdListenPort = 8001;
     static int reconnectMaxRetry = 5;
     static int readTimeoutSeconds = 90;
+    public static int defaultBufferSize = 64 * 1024;
```

Comment:
@vishesh92 somehow the websocket pkg max size has dropped to 16kB? Can you check what changed?

Reply:
checking